test/services/sync/strategies/mixins/ClassImporter.test.js

```diff
@@ -0,0 +1,261 @@
+const { expect } = require('chai');
+const { mix } = require('mixwith');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mixwith'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)